recipes-ni/initscripts-nilrt/files/nisdbootconfig-safe

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+# check if target is NOT using ubifs (mtd)
+if ! cat /proc/mtd | grep -qs mtd ; then
+
+    # configure the partitions to mount by label in fstab
+    sed -i '/ubi/d' /etc/fstab
+    echo LABEL=nibootfs         /boot                ext4       sync           0  0 >>/etc/fstab
+    echo LABEL=niconfig         /etc/natinst/share   ext4       sync           0  0 >>/etc/fstab
+    echo LABEL=nirootfs         /mnt/userfs          ext4       defaults       0  0 >>/etc/fstab
+    mkdir -p /etc/natinst/share
+    mkdir -p /mnt/userfs
```

Comment:
Is the only difference between the safemode and runmode version of this script: whether or not the `nirootfs` partition is switched to `ext4`?

If that's the case, can we just have one version of the script for both, which checks to see if its in safemode and, if it is, fixes the rootfs entry. Something like

```bash
if [ -r '/etc/natinst/safemode' ]; then
    echo LABEL=nirootfs...
    mkdir -p /mnt/userfs
fi
```

Then we only have to maintain a single package and source for the script.

Reply:
Yeap, that's the only difference. I think this is a great idea. I will remove the one I dedicated for safe mode (nisdbootconfig-safe) and just maintain one version in the new recipe (nisdbootconfig), then use the safemode and runmode recipes to include this nisdbootconfig recipe.